templates/Web/_composition/Vue/Page.Vue.AddHome/.template.config/template.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+    "author": "Microsoft Community",
+    "classifications": [
+      "Universal"
+    ],
+    "name": "Page.Vue.AddHome",
+  "tags": {
```

Comment:
Same question about indentation

Reply:
Yup you're right I should fix this.

Reply:
Actually I noticed this is an issue for almost all of my template files in the repository. Do you think we should make a PR fixing formatting of all of these. I usually turn format on save off for templating because it can actually break certain templates (removes macros or edits certain lines). But we can make a giant PR fixing all template.jsons if you wish (which I think would be the better option (since this does not affect users)). Your call here though because I can fix it for all pages PRs right now.